backend/api-service/src/backend_tester.go

```diff
@@ -124,6 +127,48 @@ func (bt *backendTester) testDeleteLocationNotFound(t *testing.T) {
 	}
 }
 
+func (bt *backendTester) testDeleteAlert(t *testing.T) {
+	ctx := context.Background()
+	id := "alert-id"
+	alert := Alert{
+		Id:            "id-to-be-replaced-by-uuid",
```

Comment:
nit: technically this won't be replaced by a uuid as it's part of the initial backend state

Reply:
Changed to just "id".